cocos/scene-graph/node.ts

```diff
@@ -1749,6 +1749,14 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
         return this._layer;
     }
 
+    /**
+     * @zh 节点的变换改动版本号。
+     * @en The transformation change version number of the node.
+     */
+    get flagChangedVersion () {
```

Comment:
Why need this value? As it is used internally, if the value is returned, then can not change to mechanism in future.

Reply:
In some scenarios we need to fetch the value and cache it, and compare the current value with the cached value when needed to calculate whether there has been a change in that time.

Reply:
Could you please describe the scenarios that need it as there is `Node.hasChangedFlags()`;

Reply:
`Node.hasChangedFlags` can only be used to distinguish if there is a change in this frame, if I need to know if a change has occurred in a certain period of time, I need to call Node.hasChangedFlags once for each frame in that period of time. 
With this new interface, I only need to record version once before that time, and again after that time, and compare the following two, if they don't match, then a change has occurred, which is much better and avoids many special cases

Reply:
Got it, can i know what module need it? As it is implemented depended, so it is better to public it as possible.

Reply:
If so, it is better to mark it as engine internal.

Reply:
marked as engine internal